FRANKDOC.md

```diff
@@ -9,6 +9,7 @@ The Frank!Doc provides reference information for Frank developers as explained i
 | JavaDoc tag | `@ff.parameters` | Class | Describes how parameters (Frank config `<Param>`) are used. |
 | JavaDoc tag | `@ff.parameter` | Class | Describes the meaning of a specific parameter. First argument is name of parameter. Second argument is description of that parameter.|
 | JavaDoc tag | `@ff.forward` | Class | Describes a forward (e.g. `success`, `failure`). First argument is name of forward. Second argument is description.|
+| JavaDoc tag | `@ff.tag` | Class | Add tag for webapp. First argument is tag name, second argument is tag value.|
```

Comment:
Het is niet een tag voor de webapp, maar een tag die het element classificeert.

Reply:
Gedaan.